packages/ckeditor5-restricted-editing/src/restrictededitingmodeui.ts

```diff
@@ -75,6 +73,32 @@ export default class RestrictedEditingModeUI extends Plugin {
 			return dropdownView;
 		} );
 
+		editor.ui.componentFactory.add( 'menuBar:restrictedEditing', locale => {
```

Comment:
Clicking buttons does not close the menu. Was this your intention?

https://github.com/ckeditor/ckeditor5/assets/1099479/e1d62317-260c-4839-babd-d04248dfc0a1

Reply:
I thought it made sense in the context of navigation buttons. However, for the sake of consistency, I will fix it.